README.md

```diff
@@ -1303,3 +1538,169 @@ To close out a communication session with the head unit, an application sends an
+
+## 7. Secured Communication
+
+It is possible to establish a secured and encrypted communication with the system by setting the frame header encryption flag to `1` when starting a new service or by sending another `StartService` with the encryption flag set to `1` when the service is already established (this the required flow for the RPC service). If the authentication is successful, the system will reply with a `StartService ACK` frame with the encryption flag also set to `1` indicating that encrypted data is now accepted. If the authentication fails for some reason, the system will reset the TLS connection and return a `StartService NAK` frame.
+
+Below are possible combinations of the service encryption status and RPCs protection flag value.
+
+<table width="100%">
+  <tr>
+    <th width="30%">Service Encryption Status</th>
+    <th width="10%">RPC Type</th>
+    <th width="10%">Requires Protection</th>
+    <th width="50%">Expected SDL Behavior</th>
+  </tr>
+  <tr>
+    <td rowspan="4">Encryption is not established</td>
+    <td rowspan="2">Request</td>
+    <td>yes</td>
+    <td>SDL Core rejects the request with result code `ENCRYPTION_NEEDED` (please see policy updates for which RPCs need protection).</td>
+  </tr> 
+  <tr>
+    <td>no</td>
+    <td>SDL Core continues processing the RPC request.</td>
+  </tr>
+  <tr>
+    <td rowspan="2">Notification</td>
+    <td>yes</td>
+    <td>SDL Core does not send the notification.</td>
+  </tr> 
+  <tr>
+    <td>no</td>
+    <td>SDL Core sends the notification unencrypted.</td>
+  </tr>
+  
+  <tr>
+    <td rowspan="4">Encryption is established</td>
+    <td rowspan="2">Request</td>
+    <td>yes</td>
+    <td>
+      If unencrypted, SDL Core rejects the request with an unencrypted response and result code `ENCRYPTION_NEEDED`.<br>
+      If encrypted, SDL Core continues processing the request and sends an encrypted response.
+    </td>
+  </tr>
+  <tr>
+    <td>no</td>
+    <td>
+      If unencrypted, SDL Core continues processing the request and sends an unencrypted response.<br>
+      If encrypted, SDL Core continues processing the request and sends an encrypted response.
+    </td>
+  </tr>
+  <tr>
+    <td rowspan="2">Notification</td>
+    <td>yes</td>
+    <td>SDL Core sends the notification encrypted.</td>
+  </tr>
+  <tr>
+    <td>no</td>
+    <td>SDL Core sends the notification unencrypted.</td>
+  </tr>
+
+</table>
+
+### 7.1 Authentication
+
+The authentication is done using TLS handshake. The TLS handshake process is defined by TLS and is not part of the SDL protocol. 
+
+The below diagram shows the sequence of how the TLS handshake exchanges certificates to compute the master secret.
+
+![TLS Handshake activity diagram](https://user-images.githubusercontent.com/5848997/122258220-cb8b7100-ce9e-11eb-9b2a-a6194b0d1b68.png)
+
+Please see [SDL Overview Guides](https://github.com/smartdevicelink/sdl_overview_guides/tree/develop/docs/Protected%20Services/index.md) for more details.
```

Comment:
```suggestion
Please see [SDL Overview Guides](https://smartdevicelink.com/en/guides/pull_request/sdl-overview-guides/security/protected-services/) for more details.
```
One more link fix after the latest changes to the Overview Guides PR

Reply:
@jacobkeeler applied in https://github.com/smartdevicelink/protocol_spec/pull/41/commits/4823c337e15e4dcbf5e5918bec884f1cff192c06